js/modules/WebAPIProvider/AuthAPI.js

```diff
@@ -1,482 +1,476 @@
-        token(null);
-        subject(null);
-        permissions(null);
-    };
-
-    var api = {
-        token: token,
-        subject: subject,
-        tokenExpirationDate: tokenExpirationDate,
-        tokenExpired: tokenExpired,
-        setAuthParams: setAuthParams,
-        resetAuthParams: resetAuthParams,
-        getAuthorizationHeader: getAuthorizationHeader,
-        handleAccessDenied: handleAccessDenied,
-        refreshToken: refreshToken,
-
-        isAuthenticated: isAuthenticated,
-
-        isPermittedCreateConceptset: isPermittedCreateConceptset,
-        isPermittedUpdateConceptset: isPermittedUpdateConceptset,
-        isPermittedDeleteConceptset: isPermittedDeleteConceptset,
-        isPermittedReadConceptsets: isPermittedReadConceptsets,
-
-        isPermittedReadCohorts: isPermittedReadCohorts,
-        isPermittedReadCohort: isPermittedReadCohort,
-        isPermittedCreateCohort: isPermittedCreateCohort,
-        isPermittedCopyCohort: isPermittedCopyCohort,
-        isPermittedUpdateCohort: isPermittedUpdateCohort,
-        isPermittedDeleteCohort: isPermittedDeleteCohort,
-        isPermittedGenerateCohort: isPermittedGenerateCohort,
-        isPermittedReadCohortReport: isPermittedReadCohortReport,
-
-        isPermittedReadJobs: isPermittedReadJobs,
-
-        isPermittedEditConfiguration: isPermittedEditConfiguration,
-        isPermittedEditSourcePriority: isPermittedEditSourcePriortiy,
-
-        isPermittedReadRoles: isPermittedReadRoles,
-        isPermittedReadRole: isPermittedReadRole,
-        isPermittedEditRole: isPermittedEditRole,
-        isPermittedCreateRole: isPermittedCreateRole,
-        isPermittedDeleteRole: isPermittedDeleteRole,
-        isPermittedEditRoleUsers: isPermittedEditRoleUsers,
-        isPermittedEditRolePermissions: isPermittedEditRolePermissions,
-
-        isPermittedReadIRs: isPermittedReadIRs,
-        isPermittedCreateIR: isPermittedCreateIR,
-        isPermittedEditIR: isPermittedEditIR,
-        isPermittedDeleteIR: isPermittedDeleteIR,
-
-        isPermittedReadEstimations: isPermittedReadEstimations,
-        isPermittedReadEstimation: isPermittedReadEstimation,
-        isPermittedCreateEstimation: isPermittedCreateEstimation,
-
-        isPermittedReadPlps: isPermittedReadPlps,
-        isPermittedReadPlp: isPermittedReadPlp,
-        isPermittedCreatePlp: isPermittedCreatePlp,
-        isPermittedDeletePlp: isPermittedDeletePlp,
-
-        isPermittedSearch: isPermittedSearch,
-        isPermittedViewCdmResults: isPermittedViewCdmResults,
-        isPermittedViewProfiles: isPermittedViewProfiles,
-        isPermittedViewProfileDates: isPermittedViewProfileDates,
-
-        isPermittedReadSource: isPermittedReadSource,
-        isPermittedCreateSource: isPermittedCreateSource,
-        isPermittedEditSource: isPermittedEditSource,
-        isPermittedDeleteSource: isPermittedDeleteSource,
-    };
-
-    return api;
+	var $ = require('jquery');
```

Comment:
Let's rollback formatting changes for the file - otherwise, it is very hard to see what was changed from logical perspective